src/js/angular/core/services/jwt-auth.service.js

```diff
@@ -278,30 +269,23 @@ angular.module('graphdb.framework.core.services.jwtauth', [
                 }
             };
 
-            this.setAuthHeaders = function () {
-                return new Promise(resolve => {
-                    const auth = this.auth ? this.auth : undefined;
-                    $http.defaults.headers.common['Authorization'] = auth;
-                    // Angular doesn't send this header by default and we need it to detect XHR requests
+            this.setAuthHeaders = function() {
+                return new Promise((resolve) => {
+                    // Angular doesn't send this header by default, and we need it to detect XHR requests
                     // so that we don't advertise Basic auth with them.
                     $http.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
```

Comment:
I am not sure, but I think that this header can be removed too, because the header is added in `unauthorized.interceptor.js`.

Reply:
Yes, I removed it from here along with the method